tests_e2e/pipeline/scripts/delete_resource_groups.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
I think we will need to move the cleanup code to somewhere in LISA, otherwise developer runs are not cleaned up